README.md

```diff
@@ -2,204 +2,213 @@
 
 ## Learning Goals
 
-- Understand the CSV format
-- Work with the CSV format in Java
+- Understand the CSV format.
+- Work with the CSV format in Java.
 
-## CSV Format
+## What is a CSV?
 
-A "CSV" is a "Comma Separated Values" file, where each line is a grouping of
-properties and each entry on each line is separated by a comma to indicate the
-end of one property and the start of another one.
+Now that we can read and write text files, let's look at a different kind of
+file: a CSV. **Comma Separated Values (CSV) files** take on a format where each
+line is a grouping of properties and each entry on each line is separated by a
+comma to indicate the end of one property and the start of another.
 
-Let's consider a `Person` class:
+To explain a CSV file format better, let us revisit our students from the
+`Student` class we looked at earlier!
 
-```java
-public class Person {
-    private String firstName;
-    private String lastName;
-    private int birthYear;
-    private int birthMonth;
-    private int birthDay;
-
-    public Person(String firstName, String lastName, int birthYear, int birthMonth, int birthDay) {
-        this.firstName = firstName;
-        this.lastName = lastName;
-        this.birthYear = birthYear;
-        this.birthMonth = birthMonth;
-        this.birthDay = birthDay;
-    }
-
-    public String toString() {
-        StringBuffer personString = new StringBuffer();
-        personString.append("firstName = " + firstName);
-        personString.append("\n");
-        personString.append("lastName = " + lastName);
-        personString.append("\n");
-        personString.append("Birth date = ");
-        personString.append(birthMonth + "/");
-        personString.append(birthDay+ "/");
-        personString.append(birthYear);
-        personString.append("\n");
-
-        return personString.toString();
-    }
-
-    public String getFirstName() {
-        return firstName;
-    }
-
-    public void setFirstName(String firstName) {
-        this.firstName = firstName;
-    }
-
-    public String getLastName() {
-        return lastName;
-    }
-
-    public void setLastName(String lastName) {
-        this.lastName = lastName;
-    }
+```plaintext
+firstName,lastName,letterGrade
+Suzie,Bingham,A
+Dustin,Henderson,B
+Mike,Wheeler,C
+```
 
-    public int getBirthYear() {
-        return birthYear;
-    }
+Notice in the above, every line is comma-separated and every row is a represents
```

Comment:
every row is a represents

every row represents

Reply:
Thanks for catching this!